test_fib fails after update to BRCM SAI 4.2

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/Azure/SONiC/wiki#report-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support can be found at the following locations:

- SONiC Support Forums - https://groups.google.com/forum/#!forum/sonicproject

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**
test_fib.py was working up until we cut over to SAI 4.2 in brcm master image 436. Now it is failing consistently. We suspect this is due to a regression between 3.7 and 4.2.

**Steps to reproduce the issue:**
1. Run test_fib.py against any brcm master image from version 436 or later.

**Describe the results you received:**
test_basic_fib:
```
"======================================================================", 
E                   "FAIL: fib_test.FibTest", 
E                   "----------------------------------------------------------------------", 
E                   "Traceback (most recent call last):", 
E                   "  File \"ptftests/fib_test.py\", line 351, in runTest", 
E                   "    self.check_ip_ranges()", 
E                   "  File \"ptftests/fib_test.py\", line 154, in check_ip_ranges", 
E                   "    self.check_ip_range(ip_range, next_hop, ipv4)", 
E                   "  File \"ptftests/fib_test.py\", line 169, in check_ip_range", 
E                   "    self.check_ip_route(src_port, ip_range.get_first_ip(), exp_port_list, ipv4)", 
E                   "  File \"ptftests/fib_test.py\", line 191, in check_ip_route", 
E                   "    res = self.check_ipv4_route(src_port, dst_ip_addr, dst_port_list)", 
E                   "  File \"ptftests/fib_test.py\", line 250, in check_ipv4_route", 
E                   "    return verify_packet_any_port(self, masked_exp_pkt, dst_port_list)", 
E                   "  File \"/usr/lib/python2.7/dist-packages/ptf/testutils.py\", line 2533, in verify_packet_any_port", 
E                   "    % (result.port, device_number, ports, result.format()))", 
E                   "AssertionError: Received expected packet on port 1 for device 0, but it should have arrived on one of these ports: [27]."
```

test_hash (ipv4 & ipv6):
```
"======================================================================", 
E                   "FAIL: hash_test.HashTest", 
E                   "----------------------------------------------------------------------", 
E                   "Traceback (most recent call last):", 
E                   "  File \"ptftests/hash_test.py\", line 264, in runTest", 
E                   "    self.check_hash(hash_key)", 
E                   "  File \"ptftests/hash_test.py\", line 87, in check_hash", 
E                   "    self.check_balancing(next_hop.get_next_hop(), hit_count_map)", 
E                   "  File \"ptftests/hash_test.py\", line 252, in check_balancing", 
E                   "    assert result", 
E                   "AssertionError", 
E                   "", 
E                   "----------------------------------------------------------------------", 
E                   "Ran 1 test in 47.199s"
```

**Describe the results you expected:**
All three test cases should pass.

--- a/platform/broadcom/sai.mk
+++ b/platform/broadcom/sai.mk
@@ -1,8 +1,8 @@
-BRCM_SAI = libsaibcm_4.2.1.5-4_amd64.deb
-$(BRCM_SAI)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.2/master/libsaibcm_4.2.1.5-4_amd64.deb?sv=2015-04-05&sr=b&sig=dHfQf95FxgLBlnrGJwv6pIkAYAc6mOGbz9vvUMfxFvg%3D&se=2034-07-28T05%3A53%3A21Z&sp=r"
-BRCM_SAI_DEV = libsaibcm-dev_4.2.1.5-4_amd64.deb
+BRCM_SAI = libsaibcm_4.2.1.5-6_amd64.deb
+$(BRCM_SAI)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.2/master/libsaibcm_4.2.1.5-6_amd64.deb?sv=2015-04-05&sr=b&sig=%2BZXgixWQWbDP5gEEID4wmkAT2vB%2FW%2BAiDIHs9z1FuZQ%3D&se=2034-08-18T02%3A06%3A57Z&sp=r"
+BRCM_SAI_DEV = libsaibcm-dev_4.2.1.5-6_amd64.deb
 $(eval $(call add_derived_package,$(BRCM_SAI),$(BRCM_SAI_DEV)))
-$(BRCM_SAI_DEV)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.2/master/libsaibcm-dev_4.2.1.5-4_amd64.deb?sv=2015-04-05&sr=b&sig=n98j842RZy%2BwQh4CtOku7jz6wgV%2BusbTl9NkeRndBrE%3D&se=2037-01-13T05%3A54%3A34Z&sp=r"
+$(BRCM_SAI_DEV)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.2/master/libsaibcm-dev_4.2.1.5-6_amd64.deb?sv=2015-04-05&sr=b&sig=fGd05XpZ%2FuIJ6UP7Cr39Ctz8%2Bnh4e6O6gqXFdgOTok0%3D&se=2034-08-18T02%3A06%3A24Z&sp=r"
 
 SONIC_ONLINE_DEBS += $(BRCM_SAI)
 $(BRCM_SAI_DEV)_DEPENDS += $(BRCM_SAI)

